src/components/Input/DropDown/DropDownGroup.js

```diff
@@ -179,13 +181,13 @@ class DropDownGroup extends React.Component {
   };
 
   displayLabel = selected => {
-    const { placeholder, label } = this.props;
+    const { placeholder, label, hiddenLabel } = this.props;
 
     if (placeholder.length > 0 && selected.length === 0) {
       return placeholder;
     }
 
-    if (selected.length > 0 && label.length > 0) {
+    if (!hiddenLabel && selected.length > 0 && label.length > 0) {
```

Comment:
Please update proper test cases for these changes.

Reply:
added